### PR TITLE
Use rover groups for usir access

### PR DESCRIPTION
**Pull request by accorvin**

### CODE REVIEW

File: kfdefs/base/trino/trino-group-mapping.properties [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+telemeter-manual-approval:ableisch,acavalla,acomabon,aelganzo,agabriel,agogala,ahoness,akaplan,annelson,anowak,antgarci,aputtur,arnaik,bfurtado,bharring,bpaget,btroutma,bward,cgoern,ckeller,ckoep,clichybi,cmedeiro,cwawak,dbaker,ddelcian,dkaylor,dmick,dmoessne,dporter,dyocum,ebrand,elalance,fcanogab,fgleizes,fgrosjea,fisantos,gferrazs,glabows,gtedorst,gvaughn,gvillani,herrmann,hrivero,hukhan,ifont,igarciam,jclaretm,jdivenut,jgiordan,jholzer,jhowell,jjochum,jortizpa,jrfuller,knaeem,kperrier,lasilva,lchiaret,llopezmo,lmartinh,lnacshon,lwynter,mamburn,matsmith,mcliffor,mhicks,mjuneau,mpeterma,mrobson,mtleilia,mvardhan,mwaite,nchoudhu,ndabhi,njajodia,nreilly,parsingh,pbertera,pchriste,pescorza,pgozart,pmoogi,prangupt,pstrick,ptoraska,rbeyel,rcole,rdiazgav,riek,rking,rsandu,rsedor,rwattcha,sbelmasg,sburke,sgaikwad,sgarciam,shilim,skrenger,snetting,sosen,sreber,stauil,therman,tmanor,tmicheli,ugiordan,vnema,vpagar,weaton,wgilmore,xmorano,yulim
+>>>>>>> 73e16a6 (Use rover groups for usir access)
```
Comment (anishasthana) [inline kfdefs/base/trino/trino-group-mapping.properties:14]: Looks like a merge conflict was accidentally committed in.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: kfdefs/base/trino/trino-group-mapping.properties (hunk 1 above)
Comment (accorvin) [reply kfdefs/base/trino/trino-group-mapping.properties:14]: good catch. stand by.

Comment (accorvin) [reply kfdefs/base/trino/trino-group-mapping.properties]: @anishasthana fixed.

**Review by gmfrasca — COMMENTED** [review]
/lgtm

**Pull request merged**